docs/tutorials/subscription-platform.md

```diff
@@ -377,6 +378,72 @@ After the initial payment during subscription creation, the recurring payments f
 
 The script will make up to a configurable number of [attempts to pay](https://developer.paypal.com/docs/archive/express-checkout/ec-set-up-reference-transactions/#capture-future-payments) an invoice before cancelling the subscription.  This attempts count is saved to the invoice itself as metadata.  The invoice's metadata is also used to prevent sending multiple failed payment emails per invoice from the PayPal payment attempts.
 
+## Contentful Integration
+
+### Environments
+
+The Subscription Platform makes use of 3 Contentful [environment aliases](https://www.contentful.com/developers/docs/concepts/environment-aliases/). The environment aliases are setup to match the current supported Stripe environments, and are listed below.
+
+* Prod (master)
+  * Production environment
+  * Stripe environment: Subscription Platform
+  * Refers to the Contentful “master” environment alias.
+* Stage
+  * Stage and QA environment
+  * Stripe environment: SUB_PLAT_STAGE
+* Dev
+  * Development
+  * Stripe environment: SUB_PLAT_DEV
+
+### Deployments
+
+Deployments will be handled manually and will make use of the Merge app by Contentful, following the steps discussed in the tutorial, [Merge content type changes with Merge app](https://www.contentful.com/developers/docs/tutorials/general/merge-app/).
+
+* For each merge, ensure to complete the optional step "Export migration script" and store the migration scripts in [the FxA monorepo](https://github.com/mozilla/fxa/). (##TODO: Add URL to migration script directory)
+
+:::caution
+Changes made in the Appearance tab of a field of a content type, are not merged when using the Merge app, and therefore also not included in the migration script.
+:::
+
+#### Typical changes
+
+Changes are made in the Dev environment and follow the deployment path as shown below.
+
+```mermaid
+flowchart LR
+    Dev --> Stage --> Prod
+```
+
+For a typical change and deployment, the following steps should be taken.
+
+1. Make changes in the `Dev` environment
+1. After a regular FxA release has been deployed to Stage, using the Merge App, merge changes from the `Dev` environtment alias to the `Stage` environment alias.
+1. Similarly, after a regular FxA release has been deployed to Production, using the Merge App, merge changes from the `Stage` environtment alias to the `Prod` environment alias.
```

Comment:
- Spelling error on both lines: "environtment."
- Change "Merge App" to "Merge app" as that is what Contentful does.